### PR TITLE
Issue 8422

### PR DESCRIPTION
**Pull request by bytetwt**
## Proposed Changes

- Fixes #issue_number
- https://github.com/ohcnetwork/care_fe/issues/8422
- Change 2
- More?

@ohcnetwork/code-reviewers

## Merge Checklist

- [ ] Add specs that demonstrate bug / test a new feature.
- [ ] Update [product documentation](https://docs.ohc.network).
- [ ] Ensure that UI text is kept in I18n files.
- [ ] Prep screenshot or demo video for changelog entry, and attach it to issue.
- [ ] Request for Peer Reviews
- [ ] Completion of QA

### CODE REVIEW

File: src/Components/Common/prescription-builder/ProcedureBuilder.tsx [hunk 1]
```diff
@@ -64,7 +66,7 @@ export default function ProcedureBuilder(props: Props<ProcedureType>) {
                   </button>
                 </div>
                 <div className="w-full">
-                  Procedure
+                  {t("Procedure Suggestions")}
```
Comment (rithviknishad) [inline src/Components/Common/prescription-builder/ProcedureBuilder.tsx:69]: Key needs to be added to `en` locale files under `Consultation.json`

File: .env [hunk 2]
```diff
@@ -10,3 +10,4 @@ REACT_PUBLIC_URL=https://care.ohc.network
 ESLINT_NO_DEV_ERRORS=true
 
 CARE_CDN_URL="https://egov-s3-facility-10bedicu.s3.amazonaws.com https://egov-s3-patient-data-10bedicu.s3.amazonaws.com http://localhost:4566"
+
```
Comment (rithviknishad) [inline .env:13]:
```suggestion
```

**Review by rithviknishad — CHANGES_REQUESTED** [review]
As already mentioned in https://github.com/ohcnetwork/care_fe/pull/8442#issuecomment-2320259224

Please update the PR title and body

**Comment by nihal467** [comment]
@Adiflickzzz closing this PR as invalid

**Comment by bodhish** [comment]
@Adiflickzzz please don't spam

**Pull request closed without merging**